alloy/src/main/java/com/liferay/faces/alloy/component/selectstarrating/internal/SelectStarRatingRendererBase.java

```diff
@@ -52,14 +52,6 @@ public void encodeAlloyAttributes(FacesContext facesContext, ResponseWriter resp
 		SelectStarRating selectStarRating = (SelectStarRating) uiComponent;
 		boolean first = true;
 
-		Object value = selectStarRating.getValue();
```

Comment:
@vsingleton Please elaborate on what problem was fixed by removing this code along with the encodeValue method. Thx.